CI/appveyor.after_success.ps1

```diff
@@ -39,7 +39,13 @@ if ("$Env:APPVEYOR_REPO_TAG" -eq "false") {
   Move-Item $Env:APPVEYOR_BUILD_FOLDER\src\release\* $SQUIRRELWINBIN
 
   Write-Output "=== Creating Nuget package ==="
-  nuget pack C:\projects\installers\windows\mudlet.nuspec -Version $($Env:VERSION) -BasePath $SQUIRRELWIN -OutputDirectory $SQUIRRELWIN
+  if ($Env:MUDLET_VERSION_BUILD.StartsWith('-public-test-build'))
+    # allow public test builds to be installed side by side with the release builds
+    (Get-Content C:\projects\installers\windows\mudlet.nuspec).replace('<id>Mudlet</id>', '<id>Mudlet.PublicTestBuild</id>') | Set-Content C:\projects\installers\windows\mudlet.nuspec
+    nuget pack C:\projects\installers\windows\mudlet.nuspec -Version $($Env:VERSION) -BasePath $SQUIRRELWIN -OutputDirectory $SQUIRRELWIN
```

Comment:
Could probably put this line after the if-block as it's the same as the else branch.

Reply:
Fixed.